package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
- [ ] The html-webpack-plugin is not listed as a dependency, it should automatically be once you run npm install --save-dev html-webpack-plugin, kindly do that

Reply:
it is in line 18 of  package.json   ===>>>  "webpack": 5.65.0

Reply:
"devDependencies": {
    "css-loader": "^6.5.1",
    "html-webpack-plugin": "^5.5.0",
    "style-loader": "^3.3.1",
    "webpack": "^5.65.0",
    "webpack-cli": "^4.9.1"
  }
}

Reply:
installed html-webpack-plugin